src/fides/api/service/user/fides_user_service.py

```diff
@@ -0,0 +1,118 @@
+import uuid
+from datetime import datetime
+from typing import Tuple
+
+from loguru import logger
+from sqlalchemy.orm import Session
+
+from fides.api.api.v1.endpoints.messaging_endpoints import user_email_invite_status
+from fides.api.common_exceptions import AuthorizationError
+from fides.api.models.client import ClientDetail
+from fides.api.models.fides_user import FidesUser
+from fides.api.models.fides_user_invite import FidesUserInvite
+from fides.api.schemas.messaging.messaging import (
+    MessagingActionType,
+    UserInviteBodyParams,
+)
+from fides.api.schemas.redis_cache import Identity
+from fides.api.service.messaging.message_dispatch_service import dispatch_message
+from fides.config import FidesConfig
+from fides.config.config_proxy import ConfigProxy
+
+
+def invite_user(db: Session, config_proxy: ConfigProxy, user: FidesUser) -> None:
+    """
+    Generates a user invite and sends the invite code to the user via email.
```

Comment:
nit: i'd just make a note here in the docstring that this method is a no-op if things aren't set up for user emails - it's a major piece of the functional behavior that i'd want to know without having to parse the impl

Reply:
Good callout!